docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Spryker Cloud Commerce OS management
+description: The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service hosting for a Spryker customer.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service (PaaS) hosting for a Spryker customer.
+
+## Glossary
+
+**Responsible**: Those who do the work to complete the task.
+
+**Accountable**: The one ultimately answerable for the correct and thorough completion of the deliverable or task, and the one who delegates the work to those responsible.
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is two-way communication.
+
+**Informed**: Those who are kept up-to-date on progress, often only on completion of the task or deliverable; and with whom there is just one-way communication.
+
+
+**Application**: Customer Commerce Solution build using Spryker Commerce OS.
```

Comment:
@michalwroblewski-spryker what is a `Customer Commerce Solution`?

Reply:
@andriitserkovnyi  This is the commerce solution the customer build based on our Spryker. Eg by customizing the existing b2b demo shop.